packages/mock_server/lib/mock_server.dart

```diff
@@ -0,0 +1,161 @@
+library mock_server;
```

Comment:
Not sure about the formatting here, could you try to run `dart format` in the package?

Reply:
done